### PR TITLE
Projects Update

### PR DESCRIPTION
**Pull request by Uyadav207**
Hey 👋 maintainer.
Updated my index.md with new projects and blogs..

Done : 

- [x] 5 Projects
- [x] 3 Blogs

Kindly Accept my PR.
Thank, u for helping us.
@Uyadav207

### CODE REVIEW

File: content/participants/ttjc@Uyadav207/index.md [hunk 1]
```diff
@@ -25,73 +30,120 @@ I want mentorship.
...
+
+_stack_: `React.Js` | `Spotify's PlayBack SDK` | `Spotify's web API` | `Node.Js`
+
+_hosted link_: https://relaxify.netlify.app/
 
-_stack_: React-Native | Redux | Google API | Expo-CLI.
+_github link_: https://github.com/Uyadav207/relaxify
 
-_hosted link_: Will Post Soon
+#### 2. **TubeStore** : A YouTube Clone Application 📱
```
Comment (khusharth) [inline content/participants/ttjc@Uyadav207/index.md:69]: Hey! @Uyadav207 have you hosted these react-native applications on play store? If you have you should add a link to that here.

Comment (Uyadav207) [reply content/participants/ttjc@Uyadav207/index.md:69]: @khusharth, Not yet.
But will Update it Sooner.

Comment (khusharth) [reply content/participants/ttjc@Uyadav207/index.md:69]: Ok then :+1:

**Comment by khusharth** [comment]
@gkanishk Good to merge.

**Comment by hemant-45** [comment]
@gkanishk bro help me to join the job challenge

**Comment by gkanishk** [comment]
> @gkanishk bro help me to join the job challenge

@hemant-45 DM me on Telegram don't spam others PRs

**Pull request merged**